myst_parser/docutils_renderer.py

```diff
@@ -908,3 +897,22 @@ def __getattr__(self, name):
             raise NotImplementedError(msg).with_traceback(sys.exc_info()[2])
         msg = "{cls} has no attribute {name}".format(cls=type(self).__name__, name=name)
         raise AttributeError(msg).with_traceback(sys.exc_info()[2])
+
+
+def _dict_to_docinfo(data):
```

Comment:
I'd rename this just `dict_to_docinfo`, since the `_` denotes a private method, but obviously you'll be looking to import this in `sphinx-notebook`.

Reply:
It could also be an `@staticmethod` on the class, but that's not a deal breaker.

Reply:
@chrisjsewell but if it were a `@staticmethod` then wouldn't it trip over itself when it did all the `self.current_node` stuff?

Reply:
(also, I renamed it to not be hidden)

Reply:
Oh I meant just the `dict_to_docinfo` func as it is, but I wouldn't worry about it.